main_small_batch.py

```diff
@@ -152,7 +152,7 @@ def problem_logic_batch(m):
                     model_args={},
                     starting_point=starting_point,
                     ext_dict=ext_ref,
-                    mip_transformation=True,
+                    mip_transformation=False,
```

Comment:
I see that you removed the MIP transformation for each one of these steps. I would like you to check each subproblem, what the instance looks like, and what that MIP transformation is introducing. Start with small_batch.